sys/dev/aic7xxx/ahc_pci.c

```diff
@@ -144,20 +144,23 @@ ahc_pci_map_registers(struct ahc_softc *ahc)
 	regs_type = 0;
 	regs_id = 0;
 
+	// Default to memio
+	allow_memio = 1;
+
```

Comment:
This is better off as an initializer, imho. Also , we try not to do C++ comments. Tweaked before the push.

Reply:
Ah,.I didn't realize that initializers were allowed. I'll keep that in mind for the future, also the comment style, thanks!